docs/docs/Tutorials/ROS2/PythonTemplate.md

```diff
@@ -0,0 +1,323 @@
+# Quick Start &mdash; Writing External Controller With Github Template Repository
+---
+
+There are two github
```

Comment:
Some high-level intro of what this tutorial will enable people to do would be helpful. It reads very technical, which if someone doesn't understand the purpose, it just dives straight in.